deepcell/image_generators/image_generators_test.py

```diff
@@ -66,22 +66,26 @@ def test_no_transform(self):
         mask_transform = image_generators._transform_masks(
             mask, transform=None, data_format='channels_last')
         self.assertEqual(mask_transform.shape, (5, 30, 30, num_classes))
+        self.assertTrue(np.issubdtype(mask_transform.dtype, np.integer))
```

Comment:
Not every test here needs it, but it would be good to at least spot-check that the correct dtype is being used. e.g. if we assign K.floatx() to be 'float16', that the transforms are using it. I think all of these tests would pass if no downsizing was actually being performed.

Reply:
Good call, I added it to both the distance transforms tests since I think they are the only float outputs.